avm/ptn/authorization/role-assignment/tests/e2e/rg.common/main.test.bicep

```
@@ -1,4 +1,4 @@
-targetScope = 'subscription'
+targetScope = 'managementGroup'
 metadata name = 'Role Assignments (Resource Group)'
```

```suggestion
metadata name = 'Role Assignments (Resource Group scope)'
```
to align with other test cases